src/monster.cpp

```diff
@@ -162,6 +163,36 @@ void monster::poly(mtype *t)
     faction = t->default_faction;
 }
 
+void monster::update_check(){
+    if (type->upgrade_group == "NULL"){
+        return;
+    }
+    int current_day = calendar::turn.get_turn()/ DAYS(1);
+    int upgrade_time = type->upgrade_min * ACTIVE_WORLD_OPTIONS["MONSTER_GROUP_DIFFICULTY"];
+    //g->u.add_msg_if_player(m_debug, "Upgrade group: %s ", type->upgrade_group);
+    g->u.add_msg_if_player(m_debug, "Current:day: %d", current_day);
```

Comment:
heh, it was just a function that I was using to debug some stuff.  I copied it from vehicle.cpp(tranvesing the graph to * power).